Orabot.Core/Bot.cs

```diff
@@ -95,6 +97,31 @@ private void StartLongRunningServices()
 				Task.Run(() => service.ExecuteAsync(_cancellationToken));
 		}
 
+		private async Task OnReady()
+		{
+			await RegisterSlashCommands();
+			StartLongRunningServices();
+		}
+
+		private async Task RegisterSlashCommands()
+		{
+			// Uncomment to delete existing commands:
+			var globalCommands = await _client.GetGlobalApplicationCommandsAsync();
+			foreach (var guild in _client.Guilds)
+			{
+				var guildCommands = await guild.GetApplicationCommandsAsync();
+				//foreach (var guildCommand in guildCommands)
```

Comment:
This should probably happen on every start and then re-add the commands?

Reply:
Deleting the current 10 command takes about 25 seconds. Registering them takes another 20-25 seconds. Both operations take a few more seconds to update in Discord. If we ever switch to Global commands (as opposed to Application commands), the documentation says those take an hour to update.
I also see no reason why we'd do it on every run.

Reply:
What happens if any of the commands change? Can we detect that there are no new commands or old commands that are removed? Otherwise someone will need to remember to run this code here manually first or the changed commands won't work...?

Reply:
I am relying on doing it manually. This basically refreshes what you see in the Discord client, but for some reason I'm a little scared of having it automatic on every run.

Reply:
We wouldn't need to de- and re-register on every run if it's possible to check if the commands on startup are the same ones that are currently registered with Discord. You can resolve this if that is not possible to do (easily).

Reply:
I can easily check for new commands that need to be registered, but I can't check if any of the existing commands' Options got changed or any of its many properties, so I rather rely on the manual method for any updates that need to happen.

Reply:
Ok 🤷‍♂️